doc/locale/ja/LC_MESSAGES/reference.po

```diff
@@ -29008,6 +29008,60 @@ msgstr ""
 "入れ、テキストの最後にテキストの最後であるというマーク（ ``U+FFF0`` ）を入れ"
 "ます。"
 
+msgid "``TokenTable``"
+msgstr ""
+
+msgid ""
+"``TokenTable`` is a tokenizer which treats only known keywords as a token. "
+"The known keywords must be registered as a key in the table for "
+"``TokenTable``."
+msgstr ""
+"``TokenTable`` は既知のキーワードのみをトークンとして扱うトークナイザーです。"
```

Comment:
In my opinion, to make it clear, an example use case is needed.
e.g) White lists.

Reply:
I've added description.